setup.cfg

```diff
@@ -34,9 +34,9 @@ python_requires = >=3.7
 setup_requires =
   setuptools_scm
 install_requires =
-  astropy>=4.1
+  astropy>=4.2
   gwcs>=0.15
```

Comment:
```suggestion
  gwcs>=0.16
```

Reply:
Thinking about this a bit more, we should do it in a separate PR for 2.1 rather than a bug fix.